notebook/static/notebook/js/celltoolbarpresets/rawcell.js

```diff
@@ -5,20 +5,21 @@ define([
     'notebook/js/celltoolbar',
     'base/js/dialog',
     'base/js/keyboard',
-], function(celltoolbar, dialog, keyboard) {
+    'base/js/i18n'
+], function(celltoolbar, dialog, keyboard, i18n) {
     "use strict";
 
   var CellToolbar = celltoolbar.CellToolbar;
   var raw_cell_preset = [];
 
   var select_type = CellToolbar.utils.select_ui_generator([
-    ["None", "-"],
-    ["LaTeX", "text/latex"],
-    ["reST", "text/restructuredtext"],
-    ["HTML", "text/html"],
-    ["Markdown", "text/markdown"],
-    ["Python", "text/x-python"],
-    ["Custom", "dialog"],
+    [i18n._("None"), "-"],
+    [i18n._("LaTeX"), "text/latex"],
+    [i18n._("reST"), "text/restructuredtext"],
+    [i18n._("HTML"), "text/html"],
+    [i18n._("Markdown"), "text/markdown"],
+    [i18n._("Python"), "text/x-python"],
```

Comment:
I think these 5 (LaTeX, reST, HTML, Markdown, Python) are names that shouldn't be translated.